java/client/src/test/java/glide/api/RedisClientTest.java

```diff
@@ -4050,6 +4051,23 @@ public void type_returns_success() {
         assertEquals(value, payload);
     }
 
+    @SneakyThrows
+    @Test
+    public void randomKey() {
```

Comment:
you will also need to add tests for: 

- [ ] randomKey(Route route)
- [ ] randomKey()

in `RedisClusterClientTest`